zkevm-circuits/src/bytecode_circuit/circuit/to_poseidon_hash.rs

```diff
@@ -433,44 +433,125 @@ impl<F: Field, const BYTES_IN_FIELD: usize> ToHashBlockCircuitConfig<F, BYTES_IN
 
         let empty_hash = Value::known(POSEIDON_CODE_HASH_ZERO.to_word().to_scalar().unwrap());
 
-        layouter.assign_region(
-            || "assign bytecode with poseidon hash extension",
-            |mut region| {
-                let mut offset = 0;
-                let mut row_input = F::zero();
-                for bytecode in witness.iter() {
-                    let bytecode_offset_begin = offset;
-                    base_conf.assign_bytecode(
-                        &mut region,
-                        bytecode,
-                        challenges,
-                        &push_data_left_is_zero_chip,
-                        &index_length_diff_is_zero_chip,
-                        empty_hash,
-                        &mut offset,
-                        last_row_offset,
-                        fail_fast,
-                    )?;
+        println!("assign bytecode with poseidon hash extension");
 
-                    for (idx, row) in bytecode.rows.iter().enumerate() {
-                        // if the base_conf's assignment not fail fast,
-                        // we also avoid the failure of "NotEnoughRowsAvailable"
-                        // in prover creation (so bytecode_incomplete test could pass)
-                        let offset = bytecode_offset_begin + idx;
-                        if offset <= last_row_offset {
-                            row_input = self.assign_extended_row(
+        let assignment_type = std::env::var("ASSIGNMENT_TYPE").ok().unwrap_or_default();
+        let is_parallel_assignment = match assignment_type.as_str() {
+            "default" => false,
+            "parallel" => true,
+            &_ => todo!(),
+        };
+        println!("is_parallel_assignment: {}", is_parallel_assignment);
```

Comment:
`log::debug!` is more suitable for printing out these runtime performance related log.